public/app/plugins/datasource/azuremonitor/components/ResourcePicker/ResourcePicker.test.tsx

```diff
@@ -90,19 +90,19 @@ describe('AzureMonitor ResourcePicker', () => {
   it('should show a subscription as selected if there is one saved', async () => {
     render(<ResourcePicker {...defaultProps} resources={[singleSubscriptionSelectionURI]} />);
     await waitFor(async () => {
-      const subscriptionCheckboxes = await screen.findAllByLabelText('Dev Subscription');
-      expect(subscriptionCheckboxes.length).toBe(2);
+      expect((await screen.findAllByLabelText('Dev Subscription')).length).toBe(2);
```

Comment:
```suggestion
      expect(await screen.findAllByLabelText('Dev Subscription')).toHaveLength(2);
```
Not sure how does having `findBy*` query inside of `waitFor` make the tests pass, looks weird.